tegel/src/components/theme/colour/colour-scales.stories.js

```diff
@@ -0,0 +1,60 @@
+export default {
+  title: 'Foundations/Colour',
+  parameters: {
+    layout: 'fullscreen',
+    docs: {
+      source: {
+        code: 'closed',
+      },
+    },
+  },
+};
+
+const Template = ({ colour = 'grey' }) => {
+  let scale = {
+    grey: ['50', '100', '200', '300', '400', '500', '600', '700', '800', '846', '868', '900', '958'],
+    blue: ['50', '100', '200', '300', '400', '500', '600', '700', '800', '900'],
+    red: ['50', '100', '200', '300', '400', '500', '600', '700', '800', '900'],
+  };
+  const picked = scale[colour];
+  let div = '';
+
+  picked.forEach(num => {
+    div += `<div id="test" class="demo-wrapper" style="background-color: var(--sdds-${colour}-${num})">
+      <span>${colour}-${num}</span>
```

Comment:
I think this might be more clear/useful if we put the whole syntax of the css variable here, like `--sdds-red-200`
```suggestion
      <span>--sdds-${colour}-${num}</span>
```